sql/migrations/20211122153748_world.sql

```diff
@@ -64,7 +64,7 @@ DELETE FROM `gossip_menu_option` WHERE `menu_id` = 436 AND `id` = 1;
 -- Creature 4163
 UPDATE `gossip_menu` SET `text_id` = 4795 WHERE `entry` = 4576 AND `text_id` = 4794;
 INSERT INTO `gossip_menu` (`entry`, `text_id`) VALUES
-(4576, 4795);
+(4576, 4794);

```

Comment:
Don't you actually want to replace text id 4793 here? Cause in sniff i see 4794 and 4795 for this menu, but we currently have 4793 and 4794.

Reply:
Yes, 4795 is the default greeting for rogues not 4794 (I dont know under what condition 4794 should unlock).